src/rubicon/objc/types.py

```diff
@@ -846,6 +860,12 @@ class UIEdgeInsets(Structure):
         ("right", CGFloat),
     ]
 
+    def __repr__(self):
+        return f"<UIEdgeInsets({self.top}, {self.bottom}, {self.left}, {self.right})>"
```

Comment:
It might be helpful to identify which values are which in the repo. There's no "natural" ordering like "width x height" or "(x, y)", so being explicit that `top=1 bottom=2...` is probably the best we can do.